coalib/parsing/FilterHelper.py

```diff
@@ -0,0 +1,31 @@
+from coalib.output.printers.LogPrinter import LogPrinter
+
+from coalib.parsing.filters.LanguageFilter import LanguageFilter
+from coalib.parsing.filters.CanDetectFilter import CanDetectFilter
+from coalib.parsing.filters.CanFixFilter import CanFixFilter
+
+
+class FilterHelper:
+    all_filters = {'language': LanguageFilter,
+                   'can_detect': CanDetectFilter,
+                   'can_fix': CanFixFilter}
+
+    @staticmethod
+    def is_valid_filter(filter):
+        return filter in FilterHelper.all_filters
+
+    @staticmethod
+    def get_filtered_bears(filter, args, all_bears=None):
+        if all_bears is None:
+            from coalib.settings.ConfigurationGathering import (
+                get_all_bears)
+            all_bears = get_all_bears(LogPrinter())
+        if FilterHelper.is_valid_filter(filter):
+            filter_object = FilterHelper.all_filters[filter]()
+            return filter_object.get_filtered_bears(all_bears, args)
+        raise AttributeError(FilterHelper.get_help_message(filter))
+
+    @staticmethod
+    def get_help_message(filter):
+        return '{} is invalid filter.\nAvailable filters: {}'.format(
```

Comment:
I would rather avoid `\n`s in tracebacks, they usually distort the output layout. But up to you^^